website/src/preferences/feature-regex.md

```diff
@@ -1,4 +1,4 @@
-# feature-regex
+# Feature regex
```

Comment:
Since we go with human-readable page titles (which I like), what about naming this something like "Regex for feature branches"?

Reply:
I prefer "Feature regex", mostly because it matches "Feature sync strategy". Also, if I changed this, I would want to update the other to match, and "Sync strategy for feature branches" is reeeally unwieldy in the sidebar next to all the other 2–3-word titles.